src/Aquifer.Public.API/Endpoints/Resources/Updates/List/Response.cs

```diff
@@ -11,6 +13,7 @@ public class ResponseContent
     public required ResponseContentUpdateType UpdateType { get; set; }
 
     public required int LanguageId { get; set; }
+    public string LanguageCode { get; set; } = null!;
```

Comment:
Can `LanguageCode` be made `required` and the default `null` assignment be removed?